StPetersburg/InEtn40555.xml

```diff
@@ -0,0 +1,151 @@
+            <sourceDesc>
+              <msDesc type="scroll">
+                <msIdentifier>
+                    <repository ref="INS0524InEtn"/>
+                    <idno>4055-5</idno>
+                    <altIdentifier>
+                      <idno>Platonov cat. III.123</idno>
+                    </altIdentifier>
+                </msIdentifier>
+                <msContents>
+                  <summary>
+                  </summary>
+                  <msItem xml:id="ms_i1">
+                    <title type="incomplete" ref="LIT5690MagicPrayer"><certainty locus="value" match="../@ref" cert="medium"></certainty></title>
+                    <textLang mainLang="gez"/>
+                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"></gap> ጸሎት፡ በእንተ፡ ሕማመ፡ ዓይነ፡ ጥላ፡ ወዓይነ፡ ወርቅ፡</incipit>
+                    <note>The end of the prayer is missing.</note>
+                  </msItem>
+
+                </msContents>
+                <physDesc>
+                  <objectDesc form="Scroll">
+                      <supportDesc>
+                          <support>
+                              <material key="parchment"/>
+                              <note>The parchment is described as thin in <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                              <citedRange unit="page">220</citedRange></bibl>.</note>
+                          </support>
+                          <extent>
+                              <measure unit="leaf">1</measure>
+                              <dimensions type="outer" unit="mm">
+                                <height>630</height>
+                                <width>50</width>
+                              </dimensions>
+                          </extent>
+
+                      </supportDesc>
+                      <layoutDesc>
+                          <layout columns="1" writtenLines="18">
+                          </layout>
+                      </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                    <handNote script="Ethiopic" xml:id="h1">
+                        <seg type="ink">Black, red</seg>
+                        <date>Second half of the 19th-cent - beginning of the 20th century according to <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                        <citedRange unit="page">220</citedRange></bibl>.</date>
+                    </handNote>
+                  </handDesc>
+                  <decoDesc>
+                    <decoNote xml:id="d1" type="ornamentation">
+                        <desc>Ornamental band with three <term key="cross">crosses</term>, published in <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                        <citedRange unit="page">221</citedRange></bibl>.</desc>
+                    </decoNote>
+
+                  </decoDesc>
+
+                </physDesc>
+                <history>
+                  <origin>
+                    <origDate notBefore="1850" notAfter="1909"></origDate>
```

Comment:
you could give evidence type="lettering" (it this is true)

Reply:
done